extension-impl/remoting-bolt/src/main/java/com/alipay/sofa/rpc/filter/ConsumerGenericFilter.java

```diff
@@ -58,6 +63,12 @@ public boolean needToLoad(FilterInvoker invoker) {
     @Override
     public SofaResponse invoke(FilterInvoker invoker, SofaRequest request) throws SofaRpcException {
         try {
+
+            final String revised = (String) request.getRequestProp(REVISE_KEY);
+            //if has revised, invoke directly
+            if (StringUtils.isNotBlank(revised) && REVISE_VALUE.equals(revised)) {
```

Comment:
This `if` can be simplify to `REVISE_VALUE.equals(revised)`

Reply:
ok